shared/cfg/base.cfg

```diff
@@ -73,6 +73,13 @@ netdst = virbr0
 #advanced network configuration of the host
 #host_ip_addr = ""
 
+# For enabling virt queues in tests uncomment below two params and
+# modify based on need, net_driver value can either be vhost or qemu
+# vhost - use host kernel, qemu - userspace, bydefault it is vhost if
+# kernel supports
+# queues = 10
```

Comment:
will the param name `virt_queues` be better ?

Reply:
@balamuruhans we are reusing the param that is used by qemu tests aswell, so this should be fine for now, will work on a different PR later to change the name.

Reply:
okay